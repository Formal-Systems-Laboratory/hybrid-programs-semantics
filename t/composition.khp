vars a , b ; a := a + 1 ; b := a * 2
