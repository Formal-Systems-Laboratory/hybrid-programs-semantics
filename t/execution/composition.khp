vars a , b ; a := a + 1.0 ; b := a * 2.0
