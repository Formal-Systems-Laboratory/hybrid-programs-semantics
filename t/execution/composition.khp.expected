<generatedTop>
  <k>
    .
  </k>
  <pgmVars>
    b , a , .Ids
  </pgmVars>
  <state>
    a |-> #VarReal ( a ) + 1.0
    b |-> ( #VarReal ( a ) + 1.0 ) * 2.0
  </state>
  <evolutionConditions>
    .Set
  </evolutionConditions>
  <nonDetAssignments>
    .Map
  </nonDetAssignments>
  <counter>
    0
  </counter>
</generatedTop>
